team.json

```
@@ -18,8 +18,8 @@
     "name": "Nick Sweeting",
     "other_skills": "Product Management, Devops, Distributed Systems, UI/UX",
     "summary": "Mesh networker, internet archiving hobbyist, senior full-stack engineer, founder, public transit & bicycle fanatic.",
-    "survival_skills": "wielding the slotted aluminum plate",
-    "gadget": "wip",
+    "survival_skills": "Wielding the slotted aluminum plate",
+    "gadget": "Wip",
```

lol this was wip as in `work in progress` not a whip
```suggestion
    "survival_skills": "ADV Motorbiking, camping, mountaineering",
    "gadget": "HackRF",
```